src/parser/PropertyPath.h

```diff
@@ -17,22 +17,22 @@ class PropertyPath {
   enum class Operation {
     SEQUENCE,
     ALTERNATIVE,
-    TRANSITIVE,
-    TRANSITIVE_MIN,  // e.g. +
-    TRANSITIVE_MAX,  // e.g. *n or ?
     INVERSE,
-    IRI
+    IRI,
+    ZERO_OR_MORE,
+    ONE_OR_MORE,
+    ZERO_OR_ONE
```

Comment:
In the future, we can also refactor this class to be something like
`variant<Iri, Variable, PropertyPath>`, but this PR is large enough.